packages/components/src/components/contract-card/contract-card.tsx

```diff
@@ -40,7 +41,8 @@ const ContractCard = ({
     toggleUnsupportedContractModal,
 }: React.PropsWithChildren<TContractCardProps>) => {
     const fallback_result = profit_loss >= 0 ? 'won' : 'lost';
-
+    const payout = formatMoney(contract_info.currency ?? '', profit_loss, true, 0, 0, true);
```

Comment:
What does the last `true` represent here? I don't think there's sixth parameter in `formatMoney`.

Reply:
i added new parameter but changed it back.